### ISSUE TITLE
Compile error in compiler/generator/llvm/clang_code_container.hh

### ISSUE THREAD

**Issue by agraef** [grame-cncm/faust#1039]
Hi @sletz, while compiling the latest Faust source from HEAD as a submodule in pd-faustgen2, I'm getting this compile error:

~~~
[ 66%] Building CXX object faust/build/CMakeFiles/staticlib.dir/__/compiler/libcode.cpp.o
In file included from /home/ag/Sources/github/pd-faustgen/faust/compiler/libcode.cpp:113:
/home/ag/Sources/github/pd-faustgen/faust/compiler/generator/llvm/clang_code_container.hh:29:10: fatal error: c_code_container.hh: No such file or directory
   29 | #include "c_code_container.hh"
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
~~~

(This is on Manjaro Linux, with LLVM 18.)

The c_code_container.hh file is present in generator/c. Maybe there's some include path missing there?

**Comment by agraef** [comment]
NB: Compiler is gcc version 14.1.1 20240522 (not sure whether that matters).

**Comment by agraef** [comment]
Well, I guess that the problem is with the Faust cmake setup in faustgen2, as the stand-alone source compiles fine. Sorry for the noise.

**Comment by agraef** [comment]
Nope, looks like there actually are a few include paths missing in the FAUST_INC variable in faust/build/in CMakeLists.txt. Maybe this happens when only building the library with the LLVM backend, as we do in faustgen2? Anyway, here are the required changes:

~~~diff
diff --git a/build/CMakeLists.txt b/build/CMakeLists.txt
index 98d4fe424..06d6863cb 100644
--- a/build/CMakeLists.txt
+++ b/build/CMakeLists.txt
@@ -60,6 +60,8 @@ set (FAUST_INC ${SRCDIR}
        ${SRCDIR}/DirectedGraph
        ${SRCDIR}/interval
        ${SRCDIR}/generator
+       ${SRCDIR}/generator/c
+       ${SRCDIR}/generator/cpp
        ${SRCDIR}/generator/dlang
        ${SRCDIR}/generator/csharp
        ${SRCDIR}/generator/fir
~~~

**Comment by agraef** [comment]
Fix in #1040.
